build.gradle

```diff
@@ -714,6 +716,7 @@ uiTestTargets.each { version ->
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
       // TODO (VirtusLab/ide-probe#195): remove the need to set these env variables
+      environment 'MY_CI_SCREENSHOTS_DIR', '/build/UIException'
       environment 'IDEPROBE_TEST_SUITE', 'uiTest' + version
```

Comment:
Also! even with https://github.com/VirtusLab/ide-probe/pull/197 i.e. `0.31.0+9-882007b8-SNAPSHOT` version being used here... are these env vars still needed? 🤔 again, I'm lost a bit 😅

Reply:
Oh okay, they could be **unset** but then screenshots for all versions would land into a single directory, right?

Reply:
There are not needed now, but without them screenshots will be saved directly into build/UIException (but we want to keep versions so set `IDEPROBE_TEST_SUITE`). Working with  `0.31.0+9-882007b8-SNAPSHOT` allow us to skip` IDEPROBE_TEST_CASE` and and store directly into `MY_CI_SCREENSHOTS_DIR/IDEPROBE_TEST_SUITE` (but I leave this because we probably still working on 0.31.0)